projects/igniteui-angular/src/lib/core/selection.ts

```diff
@@ -60,7 +60,7 @@ export class IgxSelectionAPIService {
         if (sel === undefined) {
             sel = this.get_empty();
         }
-        if (!itemID) {
+        if (itemID == null) {
```

Comment:
Even though `undefined == null`, returns true, it's clear and easier to understand `itemID === null || itemID === undefined` (I've checked our source code and the second approach is used). In addition when `itemID === ''` then this will not be handled.

Reply:
Modified to check for all falsy values except 0 `!itemID && itemID !== 0`.